scripts/integration-tests/e2e-prettier.sh

```diff
@@ -40,6 +40,6 @@ yarn install
 
 # Only run js,jsx,misc format tests
 # Without --runInBand CircleCI hangs.
-yarn test "tests/format/(jsx?|misc)/" --update-snapshot --runInBand
+yarn test "tests/format/(jsx?|misc|typescript|flow|flow-repo)/" --update-snapshot --maxWorkers=2
```

Comment:
We may have to defer this change until prettier updated the snapshot.

Reply:
Yeah, I'll wait for https://github.com/prettier/prettier/pull/13065.

Reply:
Won't work... I'm updating snapshot in `next` branch

Reply:
Can you switch to `next`? We'll focus on this branch for a while. And if you do that `--maxWorkers` should be removed, because we use [`--runInBand`](https://github.com/prettier/prettier/blob/ada29a39907c70b40ee0597fc0b14dd2c9c8f728/package.json#L143), have to do that with `jest-light-runner`.

Reply:
I'm worried this will be a little slow.😕

Reply:
The speed seems fast enough!
But it seems that something that has been changed reappears.

Reply:
That seems another test. New breaking change?

Reply:
No, this is still the previous one. Is it a new test in the dev branch?

Reply:
It seems this test fails randomly, not sure the problem is...

Reply:
Can you try `yarn dedupe` after install?

Reply:
I'm facing the same issue in my PR... And that PR is so big, don't know how to debug.  [this change](https://github.com/prettier/prettier/pull/11479/files/d323c43034d0136ec9b68e541ec856228615369a..ed7bb8112969dd83f3d3e559490f2ce689c9b70c?file-filters%5B%5D=.js&show-viewed-files=true) fix it, but this was changed for acorn few days ago, and tests in our `next` branch is passing.

Reply:
I'll disable this test !

Reply:
> I'm facing the same issue in my PR... And that PR is so big, don't know how to debug. [this change](https://github.com/prettier/prettier/pull/11479/files/d323c43034d0136ec9b68e541ec856228615369a..ed7bb8112969dd83f3d3e559490f2ce689c9b70c?file-filters%5B%5D=.js&show-viewed-files=true) fix it, but this was changed for acorn few days ago, and tests in our `next` branch is passing.

I don't know what happened.😰

> I'll disable this test !

Thanks! I might also try to figure out why, but it won't necessarily work. :)

Reply:
Let's wait for the CI pass https://github.com/prettier/prettier/pull/13159

Reply:
Merged, try again!

Reply:
Doesn't seem to be the reason for that test.😳
But for now I think it has something to do with our tests and I'll try to find out why.

Reply:
So sorry for my bad memory!

This change was actually introduced two months ago at https://github.com/babel/babel/pull/14616.
This also caused our E2E test to fail at the time, but then the inexplicable error disappeared, so I wrongly remembered that the problem was fixed.
https://github.com/babel/babel/pull/14612#issuecomment-1140318911
Can you take a look at this comment?

Reply:
> Doesn't seem to be the reason for that test.

I must made some mistake, I thought it was the one..

Reply:
Let me get this clear, that one supposed to be invalid, and we format it.

I had a look at that test, it seems our `typescript` parser are correctly throwing, but `babel-ts` didn't.

Reply:
It's a recoverable exception, it doesn't `throw`, and ast is generated normally, maybe that has something to do with it?

Reply:
Yes, we recovery allow some recoverable error https://github.com/prettier/prettier/blob/ab72a2c11c806f3a8a5ef42314e291843e1b3e68/src/language-js/parse/babel.js#L183, have you changed the message code or throwing a different message code?

Reply:
https://github.com/babel/babel/blob/v7.18.9/packages/babel-parser/src/plugins/jsx/index.js#L38-L41

Maybe this? Looks like a new addition.

Reply:
Could be.

Reply:
Anyway, we can fix that test by disallowing that from recovering( make it invalid ), it's invalid in typescript parser after all.